RobotConstraints:
  min_vx: -0.1
  max_vx: 0.1
  min_vy: -0.1
  max_vy: 0.1
  min_omega: -0.7854
  max_omega: 0.7854